packages/storybook-vue/src/stories/RadioButton/RadioButton.stories.mdx

```diff
@@ -0,0 +1,30 @@
+import { action } from "@storybook/addon-actions";
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { RadioButton } from "@utrecht/component-library-vue";
+import { argTypes } from "./argTypes.ts";
+
+<Meta id="vue-radio-button" title="Vue.js Component/RadioButton" component={RadioButton} argTypes={argTypes} />
```

Comment:
`RadioButton` → `Radio Button` (and a few more occasions of this)

I think avoiding PascalCase in general documentation helps to keep it low-barrier, if the sidebar starts to look like it has code in it, it might scare a few people off 😉